Tests/XCUITests/HomePageSettingsUITest.swift

```diff
@@ -220,6 +220,30 @@ class HomePageSettingsUITests: BaseTestCase {
 //        waitForExistence(app.cells.staticTexts["Example Domain"], timeout: 3)
     }
 
+    func testRecentlyVisited() {
```

Comment:
you may need to add here a `waitForExistence(app.buttons["urlBar-cancel"], timeout: TIMEOUT)` to be sure we wait for the close url bar and it is actually clicked

Reply:
Resolved